inst/application/app.R

```diff
@@ -1,4 +1,20 @@
 
+#disease_list <- list(
```

Comment:
Is this commented out code still needed?

Reply:
nope, removed it